src/main/scala/com/thenewmotion/chargenetwork/ocpp/charger/SslContext.scala

```diff
@@ -0,0 +1,45 @@
+package com.thenewmotion.chargenetwork.ocpp.charger
+
+import java.io.FileInputStream
+import java.io.FileNotFoundException
+import java.security.KeyStore
+import javax.net.ssl.KeyManagerFactory
+import javax.net.ssl.SSLContext
+import javax.net.ssl.TrustManagerFactory
+
+object SslContext {
+  def apply(
+    keystoreFile: String,
+    keystorePassword: String
+  ): SSLContext = {
+    val password = keystorePassword.toCharArray
+    val clientCertKeyStore = {
+      val keyStore = KeyStore.getInstance(KeyStore.getDefaultType)
+      val fileInputStream = new FileInputStream(keystoreFile)
```

Comment:
The try and catch block is only there to ensure the file is closed if reading from it does not lead to the expected result. If an exception occurs during file opening, we actually want to error to show (so the user knows why the file couldn't be opened)